### PR TITLE
feat: building plan calculator screen

### PR DESCRIPTION
**Pull request by GusBaamonde**
Look: https://cdn.fromdoppler.com/doppler-ui-library/build992/plan-calculator.html

plans_calculator_screen

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/templates/plan-calculator.html [hunk 1]
```diff
@@ -101,14 +101,213 @@ <h1 class="dp-tit-plans">Plan STANDARD</h1>
...
+                            ><span class="dp-old-amount">15</span></span
+                          >
+                          <h2 class="dp-amount-large">
+                            <span class="dp-money">US$</span
+                            ><span class="dp-amount">11,25</span>
+                          </h2>
+                          <!--<h2 class="dp-amount-small">
+                          <span class="dp-money">Bs</span
+                          ><span class="dp-amount">85.728.236,58</span>
+                        </h2>-->
```
Comment (cbernat) [inline assets/templates/plan-calculator.html:198]: I thinks this whole section is referred to price, perhaps the naming of classes could be something like this:
 
```suggestion
                        <div class="dp-price--wrapper">
                          <span class="dp-price-old"
                            ><span class="dp-price-old--money">US$</span
                            ><span class="dp-price-old--amount">15</span></span
                          >
                          <h2 class="dp-price-large">
                            <span class="dp-price-large--money">US$</span
                            ><span class="dp-price-large--amount">11,25</span>
                          </h2>
                          <!--<h2 class="dp-price-small">
                          <span class="dp-price-small--money">Bs</span
                          ><span class="dp-price-small--amount">85.728.236,58</span>
                        </h2>-->
```
In this way I believe this has more meaning as component, one whole component dp-price with variations for size and modificators (after --) money and amount.

Comment (GusBaamonde) [reply assets/templates/plan-calculator.html:198]: Excellent!

Comment (GusBaamonde) [reply assets/templates/plan-calculator.html]: ready the change! 👍

File: assets/templates/plan-calculator.html [hunk 2]
```diff
@@ -183,18 +183,19 @@ <h4>Suscripción</h4>
...
+                        <!--uno -->
+                        <div class="dp-price--wrapper">
+                          <span class="dp-price-old"
+                            ><span class="dp-price-old-money">US$</span
+                            ><span class="dp-price-old-amount">15</span></span
                           >
-                          <h2 class="dp-amount-large">
-                            <span class="dp-money">US$</span
-                            ><span class="dp-amount">11,25</span>
+                          <h2 class="dp-price-large--amount">
```
Comment (cbernat) [inline assets/templates/plan-calculator.html:192]:
```suggestion
                          <h2 class="dp-price-large-amount">
```
The class `dp-price-large--amount` does not exist, either change it in the name or in the html

File: assets/scss/templates/_doppler-plus.scss [hunk 3]
```diff
@@ -413,60 +413,59 @@
...
+      .dp-price-old-amount {
         text-decoration: line-through;
         font-size: 24px;
         line-height: 30px;
       }
     }
 
     /* small */
-    .dp-amount-small {
+    .dp-price-small-amount {
```
Comment (cbernat) [inline assets/scss/templates/_doppler-plus.scss:442]:
```suggestion
    .dp-price-small {
```
Here `dp-price-small` would be the main component, and `dp-price-small-money` and `dp-price-small-amount` subcomponents.

File: assets/scss/templates/_doppler-plus.scss [hunk 4]
```diff
@@ -413,60 +413,59 @@
...
         font-size: 42px;
         line-height: 42px;
         font-weight: normal;
       }
     }
 
     /* large */
-
-    .dp-amount-large {
+    .dp-price-large--amount {
```
Comment (cbernat) [inline assets/scss/templates/_doppler-plus.scss:459]:
```suggestion
    .dp-price-large {
```
Same here, `dp-price-large` should be the main component.

File: assets/templates/plan-calculator.html (hunk 2 above)
Comment (cbernat) [inline assets/templates/plan-calculator.html:192]:
```suggestion
                          <h2 class="dp-price-large">
```

File: assets/templates/plan-calculator.html [hunk 5]
```diff
@@ -183,18 +183,19 @@ <h4>Suscripción</h4>
...
+                            <span class="dp-price-large-money">US$</span
+                            ><span class="dp-price-large-amount">11,25</span>
                           </h2>
-                          <!--<h2 class="dp-amount-small">
-                          <span class="dp-money">Bs</span
-                          ><span class="dp-amount">85.728.236,58</span>
+                          <!--<h2 class="dp-price-small-amount">
```
Comment (cbernat) [inline assets/templates/plan-calculator.html:196]:
```suggestion
                          <!--<h2 class="dp-price-small">
```

File: assets/templates/plan-calculator.html [hunk 6]
```diff
@@ -263,20 +264,23 @@ <h4>Contactos</h4>
...
+                          <span class="dp-price-old"
+                            ><span class="dp-price-old-money">Bs</span
+                            ><span class="dp-price-old-amount"
                               >85.728.236,58</span
                             ></span
                           >
-                          <h2 class="dp-amount-small">
-                            <span class="dp-money">Bs</span
-                            ><span class="dp-amount">85.728.236,58</span>
+                          <h2 class="dp-price-small-amount">
```
Comment (cbernat) [inline assets/templates/plan-calculator.html:279]:
```suggestion
                          <h2 class="dp-price-small">
```

**Comment by cbernat** [comment]
Very nice! 🚀

**Pull request merged**